docs/docsite/rst/scenario_guides/guide_azure.rst

```diff
@@ -374,11 +374,11 @@ the same directory.
 
 Control grouping using the following variables defined in the environment:
 
-* AZURE_GROUP_BY_RESOURCE_GROUP=yes
-* AZURE_GROUP_BY_LOCATION=yes
-* AZURE_GROUP_BY_SECURITY_GROUP=yes
-* AZURE_GROUP_BY_TAG=yes
-* AZURE_GROUP_BY_OS_FAMILY=yes
+* AZURE_GROUP_BY_RESOURCE_GROUP=true
```

Comment:
This is part of the legacy code. I wouldn't touch this part of the code unless I test it myself.

Reply:
done.